CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Develop Branch]
 
+### Removed
+
+- Removed extraneous 'endif()' line in Umpire CMakeLists.txt file
```

Comment:
I think this should be in "Fixed" rather than "Removed".

Reply:
Sure thing